devices.js

```diff
@@ -15012,6 +15012,25 @@ const devices = [
             return {'l1': 1, 'l2': 2, 'l3': 3};
         },
     },
+
+    // Livarnolux (Lidl) remote control dimmer
+    {
+        zigbeeModel: ['TS1001'],
+        model: 'TS1001',
```

Comment:
Fill in the real model here (probably written on the packaging), I expect it to start with HG